man/systemd-timesyncd.service.xml

```diff
@@ -29,35 +29,43 @@
-    cases are not covered by <filename>systemd-timesyncd</filename>.</para>
-
-    <para>The NTP servers contacted are determined from the global
-    settings in
-    <citerefentry><refentrytitle>timesyncd.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-    the per-link static settings in <filename>.network</filename>
-    files, and the per-link dynamic settings received over DHCP. See
-    <citerefentry><refentrytitle>systemd.network</refentrytitle><manvolnum>5</manvolnum></citerefentry>
-    for more details.</para>
+    <para><filename>systemd-timesyncd</filename> is a system service that may be used to synchronize the
+    local system clock with a remote Network Time Protocol (NTP) server. It also saves the local time to disk
+    every time the clock has been synchronized and uses this to possibly advance the system realtime clock on
+    subsequent reboots to ensure it monotonically advances even if the system lacks a battery-buffered RTC
+    chip.</para>
+
+    <para>The <filename>systemd-timesyncd</filename> service implements SNTP only. This minimalistic service
+    will step the system clock for large offsets or slowly adjust it for smaller deltas. Complex use cases
+    that require full NTP support (and where SNTP is not sufficient) are not covered by
+    <filename>systemd-timesyncd</filename>.</para>
+
+    <para>The NTP servers contacted are determined from the global settings in
+    <citerefentry><refentrytitle>timesyncd.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>, the
+    per-link static settings in <filename>.network</filename> files, and the per-link dynamic settings
+    received over DHCP. See
+    <citerefentry><refentrytitle>systemd.network</refentrytitle><manvolnum>5</manvolnum></citerefentry> for
+    further details.</para>
 
     <para><citerefentry><refentrytitle>timedatectl</refentrytitle><manvolnum>1</manvolnum></citerefentry>'s
-    <command>set-ntp</command> command may be used to enable and
-    start, or disable and stop this service.</para>
+    <command>set-ntp</command> command may be used to enable and start, or disable and stop this
+    service.</para>
 
     <para><citerefentry><refentrytitle>timedatectl</refentrytitle><manvolnum>1</manvolnum></citerefentry>'s
     <command>timesync-status</command> or <command>show-timesync</command> command can be used to show the
     current status of this service.</para>
+
+    <para><filename>systemd-timesyncd</filename> initialization delays the start of units that are ordered
+    after <filename>time-set.target</filename> (see
+    <citerefentry><refentrytitle>systemd.special</refentrytitle><manvolnum>7</manvolnum></citerefentry> for
+    details) until the local time has been updated from
+    <filename>/var/lib/systemd/timesync/clock</filename> (see below) in order to make it monotonic (see
+    above), should this be necessary — for example because no RTC device is available. It does not delay
```

Comment:
This is a long, hard-to-understand sentence. Also, the "see above" reference and the following re-iteration of the RTC thing doesn't pulls its weight in my opinion. I suggest cutting both out:
```suggestion
    <filename>/var/lib/systemd/timesync/clock</filename> (see below) in order to make it monotonic. It does not delay
```

The "in order to make it monotonic" should be enough for the reader to remember what was said 6 paragraphs above, I think.